diagnostic.md

```diff
@@ -5,20 +5,29 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+  Art School Website
+    -admissions
+    -location
+    -classes
+        -watercolor
+        -ceramics
+        -oil painting
+          -section 01
+          -section 02
+            -Tuesdays 6-pm with instructor Bob Ross
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    A component is produced using the component itself, the template for the component, and the application template. The template for the component shows how it will be rendered with html markup. The application template defines where the child template information will go. The component itself is a js file that defines how the component will behave, including some attributes styling (like hidden: true) and actions.
```

Comment:
Looks good.  The `template.hbs` is also responsible for binding data, and binding and dispatching actions.